frontend/src/views/groupmember/UpdateGroupMember.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div id = 'updateGroupMember' v-if= "showForm">
+    <form @submit.prevent="submitForm">
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.groupNumber" id="groupNumber" label="Group Number" type="text" v-model.trim="groupNumber" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.phn"  id="phn" label="Group Member's PHN" type="text" v-model="phn"/>
+        </AppCol>
+      </AppRow>
+       <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.groupMemberNumber" id="groupMemberNumber" label="Group Member Number"   type="text" v-model.trim="groupMemberNumber" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.departmentNumber" vi  id="departmentNumber" label="Department Number"   type="text" v-model="departmentNumber"/>
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppButton :submitting="submitting" mode="primary" type="submit">Submit</AppButton>
+        <AppButton @click="resetForm" mode="secondary" type="button">Clear</AppButton>
+      </AppRow>
+    </form>
+  </div>
+  <br />
+  <div id = "confirmation" v-if="updateOk">
+    <p>PHN: {{ result?.phn }}</p>  
+  </div>
+</template>
+
+<script>
+import GroupMemberService from '../../services/GroupMemberService'
+import useVuelidate from '@vuelidate/core'
+import { validateGroupNumber, validatePHN, VALIDATE_PHN_MESSAGE, VALIDATE_GROUP_NUMBER_MESSAGE, VALIDATE_OPTIONAL } from '../../util/validators'
+import { required, helpers } from '@vuelidate/validators'
+
+function validateOptional() {
+  if (this.departmentNumber !== '' || this.groupMemberNumber !== '') {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
I'm not sure this validation works quite as expected. If you don't enter either field then they both show up as required which isn't technically correct. I think this validation might be best handled manually. E.g. "At least one of Group Member or Department Number is required" See PhnInquiry.vue.